jwala-persistence/src/test/java/com/cerner/jwala/persistence/service/jvm/impl/JpaJvmPersistenceServiceImplTest.java

```diff
@@ -43,8 +43,10 @@
 
         @Bean
         public GroupPersistenceService getGroupPersistenceService() {
-            return new JpaGroupPersistenceServiceImpl(getGroupCrudService(),
-                                                      getGroupJvmRelationshipService());
+            return new JpaGroupPersistenceServiceImpl(
```

Comment:
Minor: I know you didn't format it this way since you just added a parameter but it would look nicer if it were formatted like this:

```java
return new JpaGroupPersistenceServiceImpl(getGroupCrudService(), getGroupJvmRelationshipService(), 
                                          getApplicationCrudService());
```